sherpa/models/parameter.py

```diff
@@ -721,12 +721,22 @@ def _repr_html_(self) -> str:
     __add__, __radd__ = _make_binop(np.add, '+')
     __sub__, __rsub__ = _make_binop(np.subtract, '-')
     __mul__, __rmul__ = _make_binop(np.multiply, '*')
-    __div__, __rdiv__ = _make_binop(np.divide, '/')
     __floordiv__, __rfloordiv__ = _make_binop(np.floor_divide, '//')
     __truediv__, __rtruediv__ = _make_binop(np.true_divide, '/')
     __mod__, __rmod__ = _make_binop(np.remainder, '%')
     __pow__, __rpow__ = _make_binop(np.power, '**')
 
+    numpy_binop_with_symbols = {
```

Comment:
So, you have removed `__div__` and `__rdiv__` but not `__add__` etc. Perhaps it's obvious when I go through the individual commits, but I've not done that yet (thanks to listening in on the XMM workshop)?